rootfs/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
 		golang \
 		make \
 		--no-install-recommends \
-	&& export DOCKER_REGISTRY_TAG=minio \
+	&& export DOCKER_REGISTRY_TAG=master \
```

Comment:
currently we can't as we are using the fork of the repository and my changes are on the mater...i can update a branch with the changes and use that branch if needed.